src/config/dynamic-theme-fixes.config

```diff
@@ -9722,6 +9722,19 @@ IGNORE INLINE STYLE
 
 ================================
 
+inet.se
+
+INVERT
+.h1yh8nho
+.i1gv55ha
+
+CSS
+:root {
+--svg-icon-default-color: #e8e6e3
```

Comment:
```suggestion
    --svg-icon-default-color: #e8e6e3 !important;
```

Reply:
Oh yes! I forgot the "important!" line, thanks!